README.md

```diff
@@ -30,14 +35,135 @@ And then execute:
 
 ```bash
 bundle
+bin/rails decidim_vocdoni:install:migrations
+bundle exec rails decidim_vocdoni:webpacker:install
+bin/rails db:migrate
 ```
 
 ## Contributing
 
 See [Decidim](https://github.com/decidim/decidim).
 
+### Developing
+
+To start contributing to this project, first:
+
+- Install the basic dependencies (such as Ruby and PostgreSQL)
+- Clone this repository
+
+Decidim's main repository also provides a Docker configuration file if you
+prefer to use Docker instead of installing the dependencies locally on your
+machine.
+
+You can create the development app by running the following commands after
+cloning this project:
+
+```bash
+$ bundle
+$ DATABASE_USERNAME=<username> DATABASE_PASSWORD=<password> bundle exec rake development_app
+```
+
+Note that the database user has to have rights to create and drop a database in
+order to create the dummy test app database.
+
+Then to test how the module works in Decidim, start the development server:
+
+```bash
+$ DATABASE_USERNAME=<username> DATABASE_PASSWORD=<password> bin/rails s
+```
+
+Note that `bin/rails` is a convenient wrapper around the command `cd development_app; bundle exec rails`.
+
+In case you are using [rbenv](https://github.com/rbenv/rbenv) and have the
+[rbenv-vars](https://github.com/rbenv/rbenv-vars) plugin installed for it, you
+can add the environment variables to the root directory of the project in a file
+named `.rbenv-vars`. If these are defined for the environment, you can omit
+defining these in the commands shown above.
+
+#### Webpacker notes
+
+As latest versions of Decidim, this repository uses Webpacker for Rails. This means that compilation
+of assets is required every time a Javascript or CSS file is modified. Usually, this happens
+automatically, but in some cases (specially when actively changes that type of files) you want to 
+speed up the process. 
+
+To do that, start in a separate terminal than the one with `bin/rails s`, and BEFORE it, the following command:
+
+```
+bin/webpack-dev-server
```

Comment:
Missing markdown code language and also adding the dollar sign ($) for the prompt (for consistency). 

```suggestion

```bash
$ bin/webpack-dev-server
```

Reply:
I've added the bash indicator but removed the `$` symbol. It is not practical if you wan to do copy paste actions